libraries/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -153,6 +153,44 @@ export enum EnvironmentVariableNames {
    */
   RUSH_TAR_BINARY_PATH = 'RUSH_TAR_BINARY_PATH',
 
+  /**
+   * If defined, when Rush runs any bulk or phased command in verbose mode, this line will be
+   * printed above the build output. The following variables are provided as interpolated values
+   * if desired:
+   *
+   *   - `${taskName}` - the name of the task being executed (as printed in the header)
+   *
+   * This value is useful for folding long build output in CI environments. Here are some suggested
+   * values for this environment variable:
+   *
+   *   - GitHub Actions:
+   *     - `::group::${taskName}`
+   *   - Azure Devops:
+   *     - `##[group]${taskName}`
+   *   - Travis CI:
+   *     - `travis_fold:start:${taskName}`
+   */
+  RUSH_BUILD_FOLDING_HEADER = 'RUSH_BUILD_FOLDING_HEADER',
```

Comment:
We've started moving away from the word "build" in favor of "operation" since commands and tasks aren't necessarily builds.

Reply:
:+1: Open to changing this, my reasoning is:

(1) It matches all existing `RUSH_BUILD_` settings like for cache, etc.
(2) Even for people very familiar with Rush, who can parse a paragraph about phases, tasks, and operations, they still think colloquially of "build" (as opposed to update, or install, or check, or publish, etc.). I guess my inclination is to use names like `RUSH_BUILD_` to take advantage of that natural shortcut, and explain in documentation that it applies to any bulk or phased command, rather than being "correct" and calling it `RUSH_OPERATION_`, leaving many casual readers totally adrift.

(Let me know what you think -- I suppose one rebuttal is that the only way to get people used to "operations" is by using it all over, but I don't know that we're really making a concerted effort to do that right now.)

Reply:
I can't stay I have a particularly strong opinion. @octogonz and @dmichon-msft came up with the name IIRC, so they might have some thoughts.